kernel: bump 6.1 to 6.1.67

graysky2 opened the pull request:
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.67

No patches affected.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne
Run-tested: x86/64/AMD Cezanne

graysky2 committed:
kernel: bump 6.1 to 6.1.66

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.66

All patches automatically rebased.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne
Run-tested: x86/64/AMD Cezanne

Signed-off-by: John Audia <therealgraysky@proton.me>

rockdrilla commented:
JFYI: commit https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-6.1.y&id=307a6525c82a5a1bc5364711ece92c2d2487e1ad introduces regression with wireless stack. Most notable: system becomes unresponsive when network disconnect occurs.

robimarko commented:
That should be irrelevant for OpenWrt as backports (Currently 6.5) are used for wireless networking drivers and not the main kernel directly.

graysky2 committed:
kernel: bump 6.1 to 6.1.67

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.67

NO patches affected.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne
Run-tested: x86/64/AMD Cezanne

Signed-off-by: John Audia <therealgraysky@proton.me>
